app/models/route.rb

```diff
@@ -16,7 +16,8 @@ class Route < ActiveRecord::Base
 
     def self.search(search)
         if search
-            Busstop.find(:all, :conditions => ["\'busStopName\' LIKE ?", "%#{search}%"])
+            Busstop.find(:all, :conditions => ['busStopName LIKE ?', "%#{search}%"])
```

Comment:
Line is too long. [84/80]<br>Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.